public/application.js

```diff
@@ -0,0 +1,28 @@
+$(document).ready(function(){
+  $('#searchauthor').click(function(){
+    event.preventDefault();
+
+//1. GET/quotes all quotes GET 
```

Comment:
@monogut no placeholders!!!! 

I beg you to please stop writing code **as placeholder for laters**. Get **one thing done**, then **move to the next** feature.